### ISSUE TITLE
my pc wont let me download it 

### ISSUE THREAD

**Issue by aiden123454321** [cheat-engine/cheat-engine#1908]
im trying to get cheat engine but my pc wont allow it and i cant fine how to disable smart screen and i dont know if this is a issue only for laptops but if anyone can help that would be much appreciated.

**Comment by cheat-engine** [comment]
try allowing it anyhow by going to the dowloaded files list (top right button of the browser) and in that screen rightclick and choose keep anyhow

**Comment by aiden123454321** [comment]
I did that and now it would open but a virus waring pops up every time I
open it. Is this due to me being in canada?

**Comment by cheat-engine** [comment]
yes. Microsoft and other AV's deem Canadians and people living in the usa as stupid and need to be protected against themselves.  Therefore they try to scare you wish virus warnings


You can tell your AV that it's ok to execute
